### PR TITLE
Fix for FB Creatives

### LINKED ISSUE #209: Doesn't render Facebook creatives
The version of prescribe.js doesn't render Facebook creatives as Facebook uses "custom" HTML tags for their ad placements.

These are not recognized by postscribe.js without a fix - I've did some changes to it already and found it to enable the rendering of those creatives, but there might be more code changes needed, so please check and adapt if necessary.

prescribe/src/streamReaders.js
```
   // Regular Expressions for parsing tags and attributes
-  var startTag = /^<([\-A-Za-z0-9_!]+)((?:\s+[\w\-]+(?:\s*=?\s*(?:(?:"[^"]*")|(?:'[^']*')|[^>\s]+))?)*)\s*(\/?)>/;
-  var endTag = /^<\/([\-A-Za-z0-9_]+)[^>]*>/;
+  var startTag = /^<([\-A-Za-z0-9_!:]+)((?:\s+[\w\-]+(?:\s*=?\s*(?:(?:"[^"]*")|(?:'[^']*')|[^>\s]+))?)*)\s*(\/?)>/;
+  var endTag = /^<\/([\-A-Za-z0-9_:]+)[^>]*>/;
   var attr = /(?:([\-A-Za-z0-9_]+)\s*=\s*(?:(?:"((?:\\.|[^"])*)")|(?:'((?:\\.|[^'])*)')|([^>\s]+)))|(?:([\-A-Za-z0-9_]+)(\s|$)+)/g;
   var fillAttr = /^(checked|compact|declare|defer|disabled|ismap|multiple|nohref|noresize|noshade|nowrap|readonly|selected)$/i;
```

I dont know if this is just for me or if its mergeable.

### PR DESCRIPTION
**Pull request by ayxos**
fix #209

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "prescribe",
+  "version": "1.0.1",
```
Comment (jnewman) [inline package.json]: Please remove this

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (jnewman) [inline .babelrc]: Please remove this

**Comment by ayxos** [comment]
Done

**Comment by ayxos** [comment]
What is the status of this topic?

**Comment by jnewman** [comment]
Been busy; I'll merge once github updates the branch and build passes

**Comment by jnewman** [comment]
In latest

**Pull request merged**